### PR TITLE
Update Matrix4.d.ts

### PR DESCRIPTION
**Pull request by arodic**
Fixed invert() return type

### CODE REVIEW

File: src/math/Matrix4.d.ts [hunk 1]
```diff
@@ -125,7 +125,7 @@ export class Matrix4 implements Matrix {
 	/**
 	 * Inverts this matrix.
 	 */
-	invert(): Matrix;
+	invert(): Matrix4;
```
Comment (arodic) [inline src/math/Matrix4.d.ts:128]: actually, this line and all other returning this should be like:


```TypeScript
invert(): this
```

**Comment by FMS-Cat** [comment]
It seems `Matrix4.invert` was introduced in #20611, here's the implementation:

https://github.com/mrdoob/three.js/blob/21417342f6f39af0ba1f5a8f76818358d67021d4/src/math/Matrix4.js#L486-L529

**Comment by FMS-Cat** [comment]
Confirmed `Matrix3.invert()` correctly specifies `Matrix3` as its return type @ current `dev`

https://github.com/mrdoob/three.js/blob/21417342f6f39af0ba1f5a8f76818358d67021d4/src/math/Matrix3.d.ts#L87-L90

File: src/math/Matrix4.d.ts (hunk 1 above)
Comment (FMS-Cat) [reply src/math/Matrix4.d.ts:128]: agreed. we might want to replace them later

**Review by 0b5vr — APPROVED** [review]

**Comment by Mugen87** [comment]
FYI: I have used `Matrix` as the return type since `getInverse()` was defined like that:

https://github.com/mrdoob/three.js/blob/21417342f6f39af0ba1f5a8f76818358d67021d4/src/math/Matrix4.d.ts#L299

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**